decidim-proposals/app/events/decidim/proposals/create_proposal_event.rb

```diff
@@ -2,20 +2,7 @@
 
 module Decidim
   module Proposals
-    class CreateProposalEvent < Decidim::Events::SimpleEvent
-      include Decidim::Events::AuthorEvent
-
-      private
-
-      def i18n_scope
-        return super unless participatory_space_event?
-
-        "decidim.events.proposals.proposal_created_for_space"
-      end
-
-      def participatory_space_event?
-        extra.dig(:participatory_space)
-      end
+    class CreateProposalEvent < Decidim::Proposals::PublishProposalEvent
```

Comment:
Why is this still around? This shouldn't be needed anymore, I think...

Reply:
Because if I remove it, the `<decidim-url>/notifications` page with previous notifications broke

Reply:
@oriolgual this is the error I get, if I remove `CreateProposalEvent` :

Unable to autoload constant Decidim::Proposals::CreateProposalEvent, expected /home/agusti/Dev/decidim-modB/decidim-draft-discard/decidim-proposals/app/events/decidim/proposals/create_proposal_event.rb to define it

Reply:
Where do you get this error?

Reply:
You're getting this error because you have old events in your DB with a reference to this class. The ideal path is providing a script to migrate those events to the new class name